data/pages-content/volunteers.md

```diff
@@ -7,20 +7,22 @@ subtitle: Help us have a great conference!
 
 Python owes much of its success to open source and the community surrounding it. In the same spirit, EuroPython is organized and run by volunteers from the Python community. Alas, we're only a few and need your help to make the show run as smoothly as possible. Each year we are joined by our awesome attendees as volunteers to help in running this wonderful conference. This page explains some of the ways you can help us.
 
-Being a volunteer means giving your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference. However this year we want to recognize people who dedicate a significant chunk of their time by offering free Combined (Tutorials + Conference) tickets to those who need it. (More details below!)
+Being a volunteer means giving your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference. However this year we want to support people who dedicate a significant chunk of their time by offering free Combined (Tutorials + Conference) tickets to make the conference more accessible. (More details below!)
 
 We will also have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.
 
 
 ## Things to remember
 
+  - This is in person volunteering and would require you to be present at the venue.
+  - While we strive to make the conference as accessible to everyone as possible, we cannot provide any travel or accomodation support to volunteers exclusively.
   - To enter the conference you need a ticket.
   - You can sign up to volunteer before having bought a ticket.
 
 
 ## Perks for volunteers
 
-  - A free conference ticket if you volunteer for eight hours or more!
+  - A free conference ticket if you volunteer for eight hours or more! You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket.
```

Comment:
Since we do not have a refund policy for people who want to volunteer and hold tickets, let's make that extra clear here.
@DhanshreeA @hypha - I'm not sure about the right verbiage for it tho.

Reply:
How about something like:
> You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. However please note we don't have a refund policy for people who volunteer for more than eight hours and already hold a ticket.

I can't think of a nicer and clearer way to say this.

Reply:
But we can still say something like: 
> You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. However please note we will not be able to refund you for the ticket after the conference ends, should you choose to volunteer for eight hours or more. If you have already bought a ticket and want a volunteer ticket instead, please cancel your current ticket and indicate your interest in a volunteer ticket in the form below. Please note we have a limited number of tickets, and we cannot guarantee a ticket to all the volunteers.

Reply:
Yay! Thanks for getting back to this @DhanshreeA 🤗 

Let's keep it honest!

`You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. However please note we will might not be able to refund you for the ticket should you choose to volunteer for eight hours or more. We're unable to commit to that at this point.`

Reply:
> No early bird, but latebird kicking in on 30 June. I think the sentence you proposed is quite nice already btw.

Okay so if someone cancels in hopes for a volunteer ticket and doesn't get it, they'll be able to afford the ticket even less in that case. So they'd be penalized for buying a ticket sooner either way. :(

Reply:
I'll go with your suggestion @Vaibhavs10 I am more comfortable we can't commit to any sort of refunds at this time. It'll be trickier to navigate people canceling their tickets right now and applying for volunteer tickets instead.